packages/webpack/test/scaffold.js

```diff
@@ -7,16 +7,25 @@ const { toSnapshotSync } = require('memfs/lib/snapshot')
 const { createContext, Script, runInNewContext } = require('vm')
 const { readFileSync } = require('fs')
 
-exports.scaffold = async function runWebpackBuild(webpackConfig) {
+exports.scaffold = async function runWebpackBuild(
```

Comment:
types 😄

Reply:
added types to scaffold. boyscout rule principle. will fix types in test specs another time too.